draft-ietf-tls-tls13.md

```diff
@@ -2095,9 +2094,10 @@ one extension (at least "key_share" or "pre_shared_key").
 
 server_version
 : This field will contain the lower of that suggested by the client
-  in the ClientHello and the highest supported by the server.  For
-  this version of the specification, the version is { 3, 4 }.  (See
-  {{backward-compatibility}} for details about backward compatibility.)
+  in the ClientHello and the highest supported by the server.
+  Servers MUST accept ClientHello versions higher than those supported.
```

Comment:
Done. Tweaked some of the surrounding words a bit so it didn't seem quite as repetitive.